app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
Nice work so far. However, it is difficult to sign-up and log-in to the app since the confirmation message does not appear on the screen/website but only in the terminal. A user may get stuck after entering the sign-up details not knowing what to do afterwards. Kindly configure your app so that this message may appear in the user's email or the screen for easy sign-up access. This will make your app more professional and user-friendly. :wink:

Reply:
It's working fine. thanks for the review.